envs/env-py310.yml

```diff
@@ -147,6 +149,7 @@ dependencies:
   - pypdf2
   - pyqt >=5.15.0
   - pyqtgraph
+  # - pyside6
```

Comment:
Is this meant to indicate "coming soon"?  Or should this be removed?

Reply:
Yes, it's meant it's coming soon, I've encountered some conflicts between opencv and pyside6, see #65 for an issue.